.invenio

```diff
@@ -3,15 +3,15 @@ flavour = RDM
 logfile = /logs/invenio-cli.log
 
 [cookiecutter]
-project_name = Zenodo
-project_shortname = zenodo
-project_site = zenodo.org
+project_name = Zenodo RDM
+project_shortname = zenodo-rdm
```

Comment:
This is required because othewise we are forced to have the project in a `zenodo` folder, which might collide with the current zenodo.

A more future proof option would be to "hardcode" services' names with the `container_name` attribute. This can be calculated based on the `project_shortname` at cookiecutting time. Currently, we are doing maths with compose version + hardcoded string (i.e. `<prefix_based_on_version>+_web_ui_1`). If the idea is approved I can work on it (cc @slint).